src/components/book/BookDetail.js

```diff
@@ -1,20 +1,29 @@
 import Book from './Book';
 import { useBookDetail } from '../../hooks/useBookDetail';
+import { Link, useParams } from 'react-router-dom';
 
 function BookDetail() {
-  const id = 1; // TODO: Use id from route
+  const id = useParams();
   const { book, loading, error } = useBookDetail(id);
 
   if (error)
     return (
       <>
-        <h1>Something went wrong :(</h1> <h2>{error}</h2>
+        <h1> Something went wrong :( </h1>
+        <h2>{error}</h2>
+        <link to="/books">Back</link>
       </>
     );
 
   if (loading) return <h3>Loading book...</h3>;
 
-  return <Book book={book} showDetail />;
+  return (
+    <div>
+      {' '}
+      <Book book={book} showDetail />
+      <Link to="/">Back to list of Books</Link>
```

Comment:
this link doesn't take us back to a list of books, it takes us to the landing page